### DISCUSSION TITLE
Is there a way to stop deepgram from attempting to auto correct speech? e.g if someone says sanctity instead of sanctuary

### DISCUSSION THREAD

**Discussion by readiverse-ai** [deepgram/community#510] [General help] [answered]
In a sentence like "It was a sanctuary for art enthusiasts" If someone misreads it as sanctity, could we get deepgram to return sanctity? something like a phonetic transcription

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment, marked as answer]
Hey @readiverse-ai, Deepgram won't correct speech based on the grammar/context of what is spoken. Deepgram listens to the phonetic sounds themselves and transcribes words one at a time, independent of other words.

An LLM is different because an LLM uses the context of the other words in the sentence to predict the next word.

Have you seen this behavior with Deepgram? If so, I would very much like to reproduce the behavior, so if you could attach an audio file and include the request parameters + request ID, that would be very helpful.

**Reply by readiverse-ai** [reply]
Thank you for this reply, I had a fundamental misunderstanding. Additionally I tried reproducing what I thought I saw earlier and could not reproduce. Thank you for your prompt reply.
